perfscale_regression_ci/scripts/scalability/descheduler/common_func.sh

```diff
@@ -0,0 +1,58 @@
+
+function wait_for_pod_creation() {
```

Comment:
let's move this to this script because it can be useful for other scripts as well: https://github.com/openshift/svt/blob/master/perfscale_regression_ci/scripts/common.sh

Reply:
Let's move all functions other than wait_for_descheduler_to_run and _get_descheduler_evicted_ to perfscale_regression_ci/scripts/common.sh

Reply:
I missed this comment. Check the other changes I made and I'll add this one in with your new suggestions

Reply:
Think everything else looks good!